From 38945262a82cf211080a3d2178515a6007ed6e41 Mon Sep 17 00:00:00 2001
From: Rechi <Rechi@users.noreply.github.com>
Date: Wed, 1 Aug 2018 17:00:00 +0200
Subject: [PATCH] [fix] check directory implementation for null

directory implementation is null, if the protocol isn't supported or vfs addon isn't installed
---
 xbmc/windows/GUIMediaWindow.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/xbmc/windows/GUIMediaWindow.cpp b/xbmc/windows/GUIMediaWindow.cpp
index b1672010a3eb..52ff94dde216 100644
--- a/xbmc/windows/GUIMediaWindow.cpp
+++ b/xbmc/windows/GUIMediaWindow.cpp
@@ -2195,7 +2195,7 @@ bool CGUIMediaWindow::GetDirectoryItems(CURL &url, CFileItemList &items, bool us
     }
     else if (!getItems.m_result)
     {
-      if (g_application.IsCurrentThread() && !m_rootDir.GetDirImpl()->ProcessRequirements())
+      if (g_application.IsCurrentThread() && m_rootDir.GetDirImpl() && !m_rootDir.GetDirImpl()->ProcessRequirements())
       {
         ret = false;
       }
